src/topology/instances/ennreal.lean

```diff
@@ -762,6 +762,15 @@ lemma tsum_le_of_sum_range_le {f : ℕ → ℝ} {c : ℝ} (hf : ∀ n, 0 ≤ f n
 le_of_tendsto' ((has_sum_iff_tendsto_nat_of_nonneg hf _).1
   (summable_of_sum_range_le hf h).has_sum) h
 
+/-- Lemma `tsum_lt_of_nonneg` shows that if a sequence `f` with non-negative terms is
```

Comment:
This one should also be renamed to `tsum_lt_tsum_of_nonneg`, both in the docstring and in the lemma name.

Reply:
There's little point including the name of the lemma in the docstring anyway

Reply:
Updated the name and removed it from the docstring.  Thanks for the suggestions!